service/controller/key/cluster.go

```diff
@@ -39,6 +39,7 @@ const (
 	TagCluster                 = "giantswarm.io/cluster"
 	TagClusterType             = "giantswarm.io/cluster-type"
 	TagClusterTypeControlPlane = "control-plane"
+	TagControlPlane            = "giantswarm.io/control-plane"
```

Comment:
This one can probably be dropped and we should then only make sure the stack type tag is applied to the stack and its resources using the right value `tccpn`.